src/Interpreters/ProcessList.h

```diff
@@ -223,7 +232,7 @@ class QueryStatus : public WithContext
 
     QueryStatusInfo getInfo(bool get_thread_list = false, bool get_profile_events = false, bool get_settings = false) const;
 
-    CancellationCode cancelQuery(bool kill);
+    CancellationCode cancelQuery(bool kill, CancelReason reason = CancelReason::MANUAL_CANCEL);
```

Comment:
Lest remove the var `bool kill` since it is  not used.
And make reason without default value.